lib/src/util/map_util.dart

```diff
@@ -78,6 +117,6 @@ Map<String, dynamic> newStyleFromProps(Map props) {
   return existingStyle == null ? <String, dynamic>{} : new Map.from(existingStyle);
 }
 
-SplayTreeSet _validDomProps = new SplayTreeSet()
+Set _validDomProps = new Set()
```

Comment:
Actually, I think we should use `HashSet`, since it's faster than `Set`, which is a `LinkedHashSet`; see https://github.com/Workiva/wdesk_sdk/pull/3822